### ISSUE TITLE
_forceRender() may not be defined in View when changing stories while the app is loaded

### ISSUE THREAD

**Issue by trajano** [storybookjs/react-native#626]
You just need to put in a default value in case it isn't set up yet in View.tsx just add 

_forceRerender = () => {};

instead of 

_forceRerender: () => void;

```diff
diff --git a/node_modules/@storybook/react-native/dist/index.js b/node_modules/@storybook/react-native/dist/index.js
index 78460b9..5811b7b 100644
--- a/node_modules/@storybook/react-native/dist/index.js
+++ b/node_modules/@storybook/react-native/dist/index.js
@@ -867,7 +867,7 @@ var View3 = class {
   _storyIndex;
   _setStory = () => {
   };
-  _forceRerender;
+  _forceRerender = () => {};
   _ready = false;
   _preview;
   _asyncStorageStoryId;
```

**Comment by dannyhw** [comment]
fix out on 8.3.10-alpha.2 currently

**Comment by dannyhw** [comment]
ok released the stable version after all so should be fixed in 8.3.10
